src/cacheReader/loaders/ModelLoader.js

```diff
@@ -356,7 +356,7 @@ export class ModelDefinition {
 	loadFrame(model, frame) {
 		let verticesX = [...model.vertexPositionsX];
 		let verticesY = [...model.vertexPositionsY];
-		let verticesZ = model.vertexPositionsZ.map(z => -z);
+		let verticesZ = [...model.vertexPositionsZ];
```

Comment:
@Dezinater  any comments on this? without it, the animations seem to flip the Z axis and render inside out.

Reply:
I think this does cause some issues with how non-animated models are exported. I forget exactly what this breaks but there would be 3 exports I'd test to make sure it's working correct, an animated model, a non animated model, and an Animaya animated model. If all of those export correctly without any special cases checking for their type then I think the code is correct

Reply:
🤔  i might be able to flip the coordinate inside the gltf export code only then. Or at worst, make it flip when an argument is passed in.

To be fair I haven't even tried animaya or static models yet...